Wikipedia/Code/WMFRelatedSectionBlackList.m

```diff
@@ -1,10 +1,18 @@
 
 
 #import "WMFRelatedSectionBlackList.h"
+#import "MWKList+Subclass.h"
 
 static NSString* const WMFRelatedSectionBlackListFileName      = @"WMFRelatedSectionBlackList";
 static NSString* const WMFRelatedSectionBlackListFileExtension = @"plist";
 
+@implementation MWKTitle (MWKListObject)
+
+- (id <NSCopying, NSObject>)listIndex {
+    return self;
```

Comment:
I didn't want to put it into a separate category since it isn't used anywhere else. It only pertains to MWKList and doesn't need to be explored to the outside world.Its a common to only define categories in the implementation they are used.